kqueen/models.py

```diff
@@ -376,6 +376,44 @@ class Organization(Model, metaclass=ModelMeta):
     policy = JSONField()
     created_at = DatetimeField(default=datetime.utcnow)
 
+    def is_deletable(self):
+        remaining = []
+        if User.list(self.namespace, return_objects=False):
+            all_users = User.list(None).values()
+            users = [u for u in all_users if u.namespace == self.namespace]
```

Comment:
not a blocker: I'd prefer to use more lines of code and make this code more readable.

Reply:
In this case I can't really see how to make this even more readable :D.

Reply:
This on is acceptable.